src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -392,7 +393,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </ItemGroup>
 
   <ItemGroup Condition="'$(ProduceReferenceAssembly)' == 'true'">
-    <IntermediateRefAssembly Include="$(IntermediateOutputPath)ref\$(TargetName)$(TargetExt)" Condition="'@(IntermediateRefAssembly)' == ''" />
+    <IntermediateRefAssembly Include="$(IntermediateOutputPath)refint\$(TargetName)$(TargetExt)" Condition="'@(IntermediateRefAssembly)' == ''" />
```

Comment:
This should use `TargetRefPath`?

Reply:
see above

Reply:
Shouldn't this one and the change below be under `ProduceReferenceAssemblyInOutputPath` opt-in?

Reply:
Why do you think this should be conditional? Do you know of people who reach directly into `obj\ref` today?

Reply:
We have a logic that does that. But that is not the only argument…

It would be best for users, to either preserve compat all the way (_helps not to have problems down the road_) or break it and document the change to agree on one way of doing things by default.

To get the older behavior, right now, I'd have to change atleast **three** variables. With my proposal, we'll only need to change **single** variable.